wp-tevko-deprecated-functions.php

```diff
@@ -412,3 +412,45 @@ function _wp_get_attachment_image_sizes_filter_shim( $sizes, $size, $image_src,
 	}
 }
 add_filter( 'wp_calculate_image_sizes', '_wp_get_attachment_image_sizes_filter_shim', 10, 5 );
+
+/**
+ * Check the content blob for an audio, video, object, embed, or iframe tags.
+ * This is a copy of `get_media_embedded_in_content()` in WP 4.4 in order to provide
+ * back compatibility to older versions of WordPress.
+ *
```

Comment:
The changes have been reverted in WP core so let's remove the second and third line. First line should mention image tags.